src/main/java/com/spm/vasylyshyn/repository/UserRepository.java

```diff
@@ -19,14 +19,14 @@ public interface UserRepository extends JpaRepository<User, Long> {
     Optional<User> findUserByUsername(String username);
     Optional<User> findUserByEmail(String email);
 
-    @Query("select new com.spm.vasylyshyn.dto.UserDto(u.id,u.username,u.firstName, u.lastName,u.phoneNumber,u.email,u.address) from User as u")
+    @Query("select new com.spm.vasylyshyn.dto.UserDto(u.id,u.username,u.firstName, u.lastName,u.phoneNumber,u.email,u.address,u.createdDate) from User as u")
```

Comment:
fix  space for `, u.creted date` for this file

Reply:
Fixed it